simphony/testing/abc_check_engine.py

```diff
@@ -212,6 +215,7 @@ def test_dataset_rename(self):
         # we should be able to access using the new "bar" name
         ds_bar = engine.get_dataset("bar")
         self.assertEqual(ds_bar.name, "bar")
+        engine.remove_dataset("bar")
```

Comment:
@tuopuu , I don't think we need this.   We have renamed the "foo" to "bar".   So now in line 222, we can add a dataset with the name of "foo".